chat-plugins/info.js

```diff
@@ -165,7 +165,8 @@ exports.commands = {
 	ipsearchall: 'ipsearch',
 	hostsearch: 'ipsearch',
 	ipsearch: function (target, room, user, connection, cmd) {
-		if (!target.trim()) return this.parse('/help ipsearch');
+		if (!target.trim() || !target.includes('.')) return this.parse('/help ipsearch');
+		if (target.includes(':')) return this.errorReply("Unfortunately, PS! does not currently support IPv6 at this time.");
```

Comment:
Eh, I meant that the command should allow IPv6 input. And this line will never actually get hit by a valid IP